src/main/scala/com/amazon/deequ/analyzers/Minimum.scala

```diff
@@ -55,4 +57,22 @@ case class Minimum(column: String, where: Option[String] = None)
   }
 
   override def filterCondition: Option[String] = where
+
+  @VisibleForTesting
+  private def criterion(nullBehavior: NullBehavior): Column = {
```

Comment:
Since we're duplicating code here, consider moving this to a trait such as `NullReplacingAnalyzer` or `ExtremumAnalyzer` that is implemented by `Minimum` and `Maximum` (Extremum is the general term for a max or min)